lib/gui/app/components/safe-webview.js

```diff
@@ -221,28 +220,10 @@ class SafeWebview extends react.PureComponent {
    * @example
    *
    * // In the webview
-   * console.log(JSON.stringify({
-   *   command: 'error',
-   *   data: 'Good night!'
-   * }));
-   *
-   * console.log(JSON.stringify({
-   *   command: 'log',
-   *   data: 'Hello, Mars!'
-   * }));
+   * console.log('Good night!')
    */
   static consoleMessage (event) {
-    if (!robot.isMessage(event.message)) {
-      return
-    }
-
-    const message = robot.parseMessage(event.message)
-
-    if (robot.getCommand(message) === robot.COMMAND.LOG) {
-      analytics.logEvent(robot.getData(message))
-    } else if (robot.getCommand(message) === robot.COMMAND.ERROR) {
-      analytics.logException(robot.getData(message))
-    }
+    analytics.logEvent(event.message)
```

Comment:
@Shou what was the use-case of a WebView sending robot commands? Do we still need to support something here?